Add the `osu!catch World Cup 2024: Quarterfinals Recap` news post (#11554)
ppy/osu-wiki: news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap.md

[![](link)](https://osu.ppy.sh/beatmapsets/2191197#fruits/4634752)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/29/Songs/Culprate_-_Junglist_Acid_osu_Edit.mp3">
    </audio>
</div>

[![](link)](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/179/Songs/ABSOLUTE_CASTAWAY_-_step_by_step.mp3">
    </audio>
</div>

[![](link)](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/93/Songs/AriableyeS_-_Raison_detre.mp3">
    </audio>
</div>

## Quarterfinals results

The matches this week were quite exciting, as even the teams who had an easier time in their first match in the losers' bracket had more difficult matchups in the second one, and some of the podium contenders for this year faced in the winners' bracket, resulting in several close matches. Here is a summary of them.

| Team A |  |  | Team B |
| --: | :-: | :-: | :-- |
| **Peru** ::{ flag=PE }:: | **6** | 0 | ::{ flag=NO }:: Norway |

> | Australia ::{ flag=AU }:: | **0** | -1 | ::{ flag=TH }:: Thailand | *win by default* |

Reviewer: previous posts don't have this column anyway

Thanks to that win, Chile secured themselves a spot in the final 6, a result that they didn't achieve since 2020. Canada, on the other hand, will still be competing, and I'm sure are determined to make the most out of that losers' bracket run.

### ::{ flag=JP }:: Japan vs. ::{ flag=GB }:: United Kingdom

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="NEED TO CHANGE LINK" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> Japan and the United Kingdom are both teams that we've rarely seen access the later rounds of the tournament in recent years. Japan managed to reach them last year for the first time since 2018 before being eliminated, finishing top 12, while the United Kingdom never made it past the Quarterfinals and the final 12 (at least since the introduction of the losers' bracket in 2015). This year though, these two teams are looking better than they have for a long time, and are looking to beat their previous results. Japan, getting seed 8, looked very promising from the very beginning, with several of their players improving a lot throughout the year. As for the United Kingdom, several of their veteran players started grinding the game again, and their newcomers keep on improving at a rapid pace. Both teams were sent to the losers' bracket in the previous week, but ended up winning their first Quarterfinals match convincingly, 6-1 against Sweden for Japan, and 6-0 against the Netherlands for the United Kingdom. Now meeting in an elimination match, one of these teams would end their CWC run here, despite their desire to keep on going and show what they're capable of.

Reviewer 2: ```suggestion
Japan and the United Kingdom are both teams that we've rarely seen access the later rounds of the tournament in recent years. Japan managed to reach them last year for the first time since 2018 before being eliminated and finishing at top 12, while the United Kingdom never made it past the Quarterfinals and the final 12 (at least since the introduction of the losers' bracket in 2015).

This year though, these two teams are looking better than they have for a long time, and are looking to beat their previous results. Japan, getting seed 8, looked very promising from the very beginning, with several of their players improving a lot throughout the year. As for the United Kingdom, several of their veteran players started grinding the game again, and their newcomers keep on improving at a rapid pace. Both teams were sent to the losers' bracket in the previous week, but ended up winning their first Quarterfinals match convincingly, 6–1 against Sweden, and 6–0 against the Netherlands. Now meeting in an elimination match, one of these teams would end their CWC run here, despite their desire to keep on going and show what they're capable of.
```

> Right off the bat, we see back-to-back breakpoints, setting the tone for an intense series. For the following picks, both teams would exhaust the mod brackets they're known to be strong at: Japan would go on to pick and win [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726) and [DT1](https://osu.ppy.sh/beatmapsets/2191244#fruits/4634876), in which ::{ flag=JP }:: [Molqus](https://osu.ppy.sh/users/1927193) would get his second FC on the map, a repeat of his earlier performance against Sweden. The United Kingdom would do the same but with the Hard Rock pool, securing points on [HR2](https://osu.ppy.sh/beatmapsets/2191172#fruits/4634714) and [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727). Now that both of these mod pools were exhausted because of the first pick on [HR1](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698) by Japan and the [DT2](https://osu.ppy.sh/beatmapsets/2191261#fruits/4634911) ban from the United Kingdom, they had to try something new.

Reviewer 2: ```suggestion
Right off the bat, we see back-to-back breakpoints, setting the tone for an intense series. For the following picks, both teams would exhaust the mod brackets they're known to be strong at — Japan would go on to pick and win [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726) and [DT1](https://osu.ppy.sh/beatmapsets/2191244#fruits/4634876), with ::{ flag=JP }:: [Molqus](https://osu.ppy.sh/users/1927193) getting their second FC on DT3, a repeat of their earlier performance against Sweden. The United Kingdom would do the same but with the Hard Rock pool, securing points on [HR2](https://osu.ppy.sh/beatmapsets/2191172#fruits/4634714) and [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727). Now that both of these mod pools were exhausted because of the first pick on [HR1](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698) by Japan and the [DT2](https://osu.ppy.sh/beatmapsets/2191261#fruits/4634911) ban from the United Kingdom, they had to try something new.
```

> It wasn't an issue for Japan though, who seemed to have the edge on Hidden, and would win their next two picks once again, on [HD3](https://osu.ppy.sh/beatmapsets/2191180#fruits/4634728) and [HD1](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684). The United Kingdom would go for a [MM2](https://osu.ppy.sh/beatmapsets/2191260#fruits/4634910) pick which worked, with a beautiful Nomod FC from ::{ flag=GB }:: [Matt_H](https://osu.ppy.sh/users/4067115). It was now time for the last pick of the match, having reached a 5-4 scoreline for Japan. The United Kingdom had to think about this one, and then perform. They deemed [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) to be their best option left. It has to be said though, that as NM1s tend to be in CWC, they're usually one of the easiest maps in the pool, meaning that both teams can perform on them. With nerves getting into play, backs against the wall, the United Kingdom unfortunately couldn't beat Japan on the consistency pick and ended up losing the match 6-4, with another impressive performance for the Japanese captain and veteran, ::{ flag=JP }:: [Molqus](https://osu.ppy.sh/users/1927193), FCing once again.

Reviewer 2: ```suggestion
It wasn't an issue for Japan though, who seemed to have the edge on Hidden, and would win their next two picks once again, on [HD3](https://osu.ppy.sh/beatmapsets/2191180#fruits/4634728) and [HD1](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684). The United Kingdom would go for a [MM2](https://osu.ppy.sh/beatmapsets/2191260#fruits/4634910) pick which worked, with a beautiful No Mod FC from ::{ flag=GB }:: [Matt_H](https://osu.ppy.sh/users/4067115). It was now time for the last pick of the match, having reached a 5–4 scoreline for Japan. The United Kingdom had to think about this one, and then perform. They deemed [NM1](https://osu.ppy.sh/beatmapsets/2191148#fruits/4634642) to be their best option left. It has to be said though, that as NM1s tend to be in CWC, they're usually one of the easiest maps in the pool, meaning that both teams can perform on them. With nerves getting into play, backs against the wall, the United Kingdom unfortunately couldn't beat Japan on the consistency pick and ended up losing the match 6–4, with another impressive performance for the Japanese captain and veteran, [Molqus](https://osu.ppy.sh/users/1927193), FCing once again.
```

---
layout: post
title: "osu!catch World Cup 2024: Quarterfinals Recap"
date: 2024-06-12 22:00:00 +0000
---

The osu!catch World Cup Quarterfinals concluded, and there are only 12 teams remaining. Let's take a look at everything that unfolded during this exciting weekend!

![](/wiki/Tournaments/CWC/2024/img/cwc2024-banner.jpg)

> After everything that happened, only 12 teams remain. Let's take a look at everything that unfolded during this exciting Quarterfinals week!

Reviewer 2: oops forgot to delete this
```suggestion
```

> As we reach the Semifinals, the tension rises in this year's osu!catch World Cup — more teams are facing elimination, and matches are getting closer. We finally had the first (and only so far) match of this tournament that went to tiebreaker, breaking the curse of the 6–4 World Cup, as well as several upsets and other nail-biters.

Reviewer 2: need a space under here to separate the mappool showcase header from the text
```suggestion
As we reach the Semifinals, the tension rises in this year's osu!catch World Cup — more teams are facing elimination, and matches are getting closer. We finally had the first (and only so far) match of this tournament that went to tiebreaker, breaking the curse of the 6–4 World Cup, as well as several upsets and other nail-biters.

```

Teams had different approaches to the pool this round, as each pick was played at least 5 times, with teams relying the most on [HR3](https://osu.ppy.sh/beatmapsets/2191179#fruits/4634727) and [NM2](https://osu.ppy.sh/beatmapsets/2191189#fruits/4634741), the more conventional and comfy picks of the pool for players used to tournaments.

The osu!catch enjoyers were also spoiled this week, with no less than three [osu! originals](/wiki/osu!_originals) in the mappool. It is a first for the osu!catch World Cup, which at most had two in last year's Grand Finals week. All three of them are of very different styles, so make sure to give them all a listen!

> [![](/news/2024\2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap\Culprate_-_Junglist_Acid_osu_Edit_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191197#fruits/4634752)

Reviewer 2: ```suggestion
[![](/news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap/Culprate_-_Junglist_Acid_osu_Edit_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191197#fruits/4634752)
```

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/29/Songs/Culprate_-_Junglist_Acid_osu_Edit.mp3">
    </audio>
</div>

> [![](/news/2024\2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap\ABSOLUTE_CASTAWAY_-_step_by_step_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684)

Reviewer 2: ```suggestion
[![](/news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap/ABSOLUTE_CASTAWAY_-_step_by_step_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191164#fruits/4634684)
```

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/179/Songs/ABSOLUTE_CASTAWAY_-_step_by_step.mp3">
    </audio>
</div>

> [![](/news/2024\2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap\AriableyeS_-_Raison_detre_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698)

Reviewer 2: ```suggestion
[![](/news/2024/2024-06-12-osu-catch-world-cup-2024-quarterfinals-recap/AriableyeS_-_Raison_detre_banner.jpg)](https://osu.ppy.sh/beatmapsets/2191169#fruits/4634698)
```

Both teams started by picking maps that you would expect from these rosters — Argentina, being the country with the highest ranked core roster in the tournament, opted for the faster and more difficult mechanical picks of the pool, such as [NM2](https://osu.ppy.sh/beatmapsets/2191189#fruits/4634741), [DT3](https://osu.ppy.sh/beatmapsets/2191178#fruits/4634726), and [HR2](https://osu.ppy.sh/beatmapsets/2191172#fruits/4634714), while Poland abused their strength in HD and unconventional maps, going for [NM3](https://osu.ppy.sh/beatmapsets/2191197#fruits/4634752), [HD3](https://osu.ppy.sh/beatmapsets/2191180#fruits/4634728), and [MM2](https://osu.ppy.sh/community/matches/114207882), which they even three-way FCd. Every pick was successful so far for these two teams, and now looking at the remaining maps and the 3–3 scoreline, the tension was starting to rise.

> 

Reviewer 2: ```suggestion
```
